include/chopper/layout/configuration.hpp

```diff
@@ -39,6 +44,30 @@ struct configuration
     bool force_all_binnings{false};
     bool output_statistics{false};
     bool debug{false};
+
+private:
+    friend class cereal::access;
+
+    template <class Archive>
+    void serialize( Archive & ar )
+    {
+      ar(CEREAL_NVP(data_file.string()));
+      ar(CEREAL_NVP(output_filename.string()));
+      ar(CEREAL_NVP(hll_dir.string()));
+      ar(CEREAL_NVP(t_max));
+    //   ar(CEREAL_NVP(aggregate_by_column));
+      ar(CEREAL_NVP(num_hash_functions));
+      ar(CEREAL_NVP(fp_rate));
+      ar(CEREAL_NVP(alpha));
+      ar(CEREAL_NVP(max_ratio));
+      ar(CEREAL_NVP(num_threads));
+      ar(CEREAL_NVP(estimate_union));
+      ar(CEREAL_NVP(rearrange_bins));
+      ar(CEREAL_NVP(determine_num_bins));
+      ar(CEREAL_NVP(force_all_binnings));
+      ar(CEREAL_NVP(output_statistics));
+    //   ar(CEREAL_NVP(debug)); ?
```

Comment:
Maybe, can't hurt :)

Reply:
I made the option hidden from the user since it is purely for us, so on second thought I wouldn't include it here because it raises attention to something that isn't visible on the help page.